### ISSUE TITLE
Focus indicator of active component lacks 3 to 1 contrast ratio

### ISSUE THREAD

**Issue by erik-deque** [dequelabs/cauldron#190]
The state of an ACTIVE user interface component does not have at least a 3 to 1 luminosity contrast ratio with either the inner or outer adjacent background.

**Comment by schne324** [comment]
Every issue raised on text inputs is invalid. The fields have [focus indications of both a border color change AND an inset solid box shadow](https://github.com/dequelabs/cauldron/blob/develop/packages/styles/forms.css#L25-L28). Therefore the inset box shadow meets the 3:1 focus indication requirement because it is #fdfdfe to #3c7aae.  The person who performed this audit only looked at the border color change and completely ignored the additional inset border (created by `box-shadow`)

<img width="731" alt="Screen Shot 2021-02-11 at 12 16 29 PM" src="https://user-images.githubusercontent.com/3180185/107694185-f3a26380-6c63-11eb-8e51-45a8bd5f0c77.png">
<img width="740" alt="Screen Shot 2021-02-11 at 12 16 37 PM" src="https://user-images.githubusercontent.com/3180185/107694197-f604bd80-6c63-11eb-8052-d64eaed0aef8.png">
<img width="737" alt="Screen Shot 2021-02-11 at 12 16 47 PM" src="https://user-images.githubusercontent.com/3180185/107694192-f56c2700-6c63-11eb-86a5-2c6aaf96a74e.png">
<img width="739" alt="Screen Shot 2021-02-11 at 12 16 53 PM" src="https://user-images.githubusercontent.com/3180185/107694190-f4d39080-6c63-11eb-91f7-a3b115dea28f.png">

**Comment by schne324** [comment]
Instance 2 is not an issue. The nav items have a focus indication from `#02101a` to `#d4dde0` of 6px in height and ~121px  wide which is a 13.94:1 contrast ratio. For "active" menu items we have a focus indication from `#283640` to `#d4dde0` which is a 8.99:1 contrast ratio.
    <img width="156" alt="Screen Shot 2021-02-11 at 12 30 07 PM" src="https://user-images.githubusercontent.com/3180185/107695012-e2a62200-6c64-11eb-8df8-76b3228d6e11.png">
    <img width="237" alt="Screen Shot 2021-02-11 at 12 36 22 PM" src="https://user-images.githubusercontent.com/3180185/107695730-d8385800-6c65-11eb-960b-a9a5b51c7a8b.png">
    <img width="239" alt="Screen Shot 2021-02-11 at 12 36 32 PM" src="https://user-images.githubusercontent.com/3180185/107695735-d9698500-6c65-11eb-898b-62b0a83a2388.png">
    <img width="233" alt="Screen Shot 2021-02-11 at 12 36 38 PM" src="https://user-images.githubusercontent.com/3180185/107695733-d8d0ee80-6c65-11eb-908d-dcc5cf5e925f.png">

**Comment by schne324** [comment]
Instance 6 is not valid either. The "alert bar" just uses our standard error toast styles/focus indication:

<img width="613" alt="Screen Shot 2021-02-12 at 9 08 53 AM" src="https://user-images.githubusercontent.com/3180185/107799217-02475400-6d12-11eb-8a76-79aaedbaa2fd.png">
<img width="618" alt="Screen Shot 2021-02-12 at 9 08 58 AM" src="https://user-images.githubusercontent.com/3180185/107799215-01aebd80-6d12-11eb-976c-f16a0efc56e2.png">

**Comment by schne324** [comment]
this is tracked in an extension ticket now
